pkg/scheduler/apis/config/v1beta3/default_plugins.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta3
+
+import (
+	"k8s.io/apimachinery/pkg/util/sets"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/klog/v2"
+	"k8s.io/kube-scheduler/config/v1beta3"
+	"k8s.io/kubernetes/pkg/features"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/names"
+	"k8s.io/utils/pointer"
+)
+
+// getDefaultPlugins returns the default set of plugins.
+func getDefaultPlugins() *v1beta3.Plugins {
+	plugins := &v1beta3.Plugins{
+		QueueSort: v1beta3.PluginSet{
+			Enabled: []v1beta3.Plugin{
+				{Name: names.PrioritySort},
+			},
+		},
+		PreFilter: v1beta3.PluginSet{
+			Enabled: []v1beta3.Plugin{
+				{Name: names.NodeResourcesFit},
+				{Name: names.NodePorts},
+				{Name: names.VolumeRestrictions},
+				{Name: names.PodTopologySpread},
+				{Name: names.InterPodAffinity},
+				{Name: names.VolumeBinding},
+				{Name: names.NodeAffinity},
+			},
+		},
+		Filter: v1beta3.PluginSet{
+			Enabled: []v1beta3.Plugin{
+				{Name: names.NodeUnschedulable},
+				{Name: names.NodeName},
+				{Name: names.TaintToleration},
+				{Name: names.NodeAffinity},
+				{Name: names.NodePorts},
+				{Name: names.NodeResourcesFit},
+				{Name: names.VolumeRestrictions},
+				{Name: names.EBSLimits},
+				{Name: names.GCEPDLimits},
+				{Name: names.NodeVolumeLimits},
+				{Name: names.AzureDiskLimits},
+				{Name: names.VolumeBinding},
+				{Name: names.VolumeZone},
+				{Name: names.PodTopologySpread},
+				{Name: names.InterPodAffinity},
+			},
+		},
+		PostFilter: v1beta3.PluginSet{
+			Enabled: []v1beta3.Plugin{
+				{Name: names.DefaultPreemption},
+			},
+		},
+		PreScore: v1beta3.PluginSet{
+			Enabled: []v1beta3.Plugin{
+				{Name: names.InterPodAffinity},
+				{Name: names.PodTopologySpread},
+				{Name: names.TaintToleration},
+				{Name: names.NodeAffinity},
+			},
+		},
+		Score: v1beta3.PluginSet{
```

Comment:
delta from v1beta2:

```diff
                Score: v1beta2.PluginSet{                                                          |                    Score: v1beta3.PluginSet{
                        Enabled: []v1beta2.Plugin{                                                 |                            Enabled: []v1beta3.Plugin{
                                {Name: names.NodeResourcesBalancedAllocation, Weight: pointer.In                                        {Name: names.NodeResourcesBalancedAllocation, Weight: pointer.In
                                {Name: names.ImageLocality, Weight: pointer.Int32Ptr(1)},                                               {Name: names.ImageLocality, Weight: pointer.Int32Ptr(1)},
                                {Name: names.InterPodAffinity, Weight: pointer.Int32Ptr(1)},       <
                                {Name: names.NodeResourcesFit, Weight: pointer.Int32Ptr(1)},                                            {Name: names.NodeResourcesFit, Weight: pointer.Int32Ptr(1)},
                                {Name: names.NodeAffinity, Weight: pointer.Int32Ptr(1)},           |                                    // Weight is doubled because:
                                                                                                   >                                    // - This is a score coming from user preference.
                                                                                                   >                                    {Name: names.InterPodAffinity, Weight: pointer.Int32Ptr(2)},
                                                                                                   >                                    // Weight is doubled because:
                                                                                                   >                                    // - This is a score coming from user preference.
                                                                                                   >                                    {Name: names.NodeAffinity, Weight: pointer.Int32Ptr(2)},
                                // Weight is doubled because:                                                                           // Weight is doubled because:
                                // - This is a score coming from user preference.                                                       // - This is a score coming from user preference.
                                // - It makes its signal comparable to NodeResourcesFit.LeastAll                                        // - It makes its signal comparable to NodeResourcesFit.LeastAll
                                {Name: names.PodTopologySpread, Weight: pointer.Int32Ptr(2)},                                           {Name: names.PodTopologySpread, Weight: pointer.Int32Ptr(2)},
                                {Name: names.TaintToleration, Weight: pointer.Int32Ptr(1)},        |                                    // Weight is tripled because:
                                                                                                   >                                    // - This is a score coming from user preference.
                                                                                                   >                                    // - Usage of node tainting to group nodes in the cluster is bec
                                                                                                   >                                    //       for many user workloads
                                                                                                   >                                    {Name: names.TaintToleration, Weight: pointer.Int32Ptr(3)},
                        },                                                                                                      },
                },                                                                                                      },
```